### ISSUE TITLE
Connect a Headphone to pc

### ISSUE THREAD

**Issue by ermiaetemadi** [telegramdesktop/tdesktop#3236]
### Steps to reproduce
1. Connect a Headphone to pc

### Expected behaviour
Nothing

### Actual behaviour
Telegram closed

### Configuration
**Operating system:**
Windows 7 pro
**Version of Telegram Desktop:**
1.0.29

**Comment by john-preston** [comment]
@ermiaetemadi When you launch tdesktop after it closes do you see a window about crash during the last launch?

**Comment by ermiaetemadi** [comment]
@john-preston
yes.a window opening and Telegram trying to connect to network and get the latest version.

**Comment by john-preston** [comment]
@ermiaetemadi Can you download and try from alpha version? You can get it here: https://desktop.telegram.org/changelog#alpha-version it should offer you to send a crash report when you launch it after a crazh.
